src/professionals/professional.service.ts

```diff
@@ -0,0 +1,10 @@
+import parseResumeFromPdf from '@/shared/resume-parser';
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class ProfessionalsService {
+async parseResume(fileBuffer: Buffer) {
+  const parsedResume = await parseResumeFromPdf(fileBuffer);
+  return parsedResume;
+  }
```

Comment:
take the file upload it to s3, then add it to the professional profile, but we can wait, till I finish my changes,